src/main/scala/com/gradle/internal/BuildScanTransformer.scala

```diff
@@ -0,0 +1,377 @@
+package com.gradle.internal
+
+import com.gradle.develocity.agent.sbt.api.configuration.BuildScan
+import com.gradle.develocity.agent.sbt.api.configuration.Server
+import com.gradle.internal.CiUtils._
+import com.gradle.internal.Utils._
+import java.util.Properties
+import java.net.URL
+import sbt.Logger
+
+/**
+ * Adds a standard set of useful tags, links and custom values to all build scans published.
+ */
+class BuildScanTransformer(serverConfig: Server, scalaVersions: Seq[String], logger: Logger)(implicit env: Env)
```

Comment:
Let's call this `CustomBuildScanEnhancements` to make this more consistent with the other plugins. `Transformer` is an implementation detail either way and shouldn't leak in to the name.
Once renamed back, I'd like to give this another review since it'll be easier to see differences.